hyperon_das_atomdb/adapters/redis_mongo_db.py

```diff
@@ -510,3 +541,14 @@ def clear_database(self) -> None:
             self.mongo_db[collection].drop()
 
         self.redis.flushall()
+
+    def _remove_links_not_toplevel(self, matches: list) -> list:
```

Comment:
Same comments I made for the RAM only adapter

Reply:
ok